src/components/TinderStyleVoting/PostInfoComponents/VotingFormCard.tsx

```
@@ -46,6 +50,37 @@ const VotingFormCard = ({
+	const calculateLockingPeriod = (convictionValue: number) => {
+		const lockPeriod = calculateLock(convictionValue);
+
+		if (!api || !apiReady) {
+			return 'No lockup period';
+		}
+
+		const res = api?.consts?.convictionVoting?.voteLockingPeriod;
+		const num = res?.toJSON();
+		const days = blockToDays(num, network);
+
+		if (days && !isNaN(Number(days)) && lockPeriod) {
```

Biome

[error] isNaN is unsafe. It attempts a type coercion. Use Number.isNaN instead.

See the MDN documentation for more details.
Unsafe fix: Use Number.isNaN instead.


(lint/suspicious/noGlobalIsNan)